tests/astro_tests/bossbodenheimer.dat

```diff
@@ -7,13 +7,13 @@
 #-----------------------------
 # Initial conditions variables
 #-----------------------------
-Simulation run id string                    : run_id = BB1
-Run SPH simulation                          : sim = sph
+Simulation run id string                    : run_id = BB1-MFV
+Run SPH simulation                          : sim = mfvmuscl
```

Comment:
Are we sure we want to change the default scheme in the tests? I don't have a strong opininion about this. At the moment all the tests use sph, which is a bit silly - we obviosly want to test also the meshless. But whatever we do needs to be consistent, we can't have a bunch of tests with the meshless and a bunch with sph.